docs/docs/concepts.mdx

```diff
@@ -744,14 +831,25 @@ LangChain provides a standardized interface for tool calling that is consistent
 
 The standard interface consists of:
 
-* `ChatModel.bind_tools()`: a method for specifying which tools are available for a model to call.
+* `ChatModel.bind_tools()`: a method for specifying which tools are available for a model to call. This method accepts [LangChain tools](/docs/concepts/#tools) here.
 * `AIMessage.tool_calls`: an attribute on the `AIMessage` returned from the model for accessing the tool calls requested by the model.
 
-There are two main use cases for function/tool calling:
+The following how-to guides are good practical resources for using function/tool calling:
 
 - [How to return structured data from an LLM](/docs/how_to/structured_output/)
 - [How to use a model to call tools](/docs/how_to/tool_calling/)
 
+For a full list of model providers that support tool calling, [see this table](/docs/integrations/chat/#advanced-features).
+
+#### `.with_structured_output()`
```

Comment:
thsi should maybe be more prominently featured since its our recomended way?

Reply:
Moved to be the first discussed method.